miio/integrations/vacuum/dreame/tests/test_dreamef9vacuum_miot.py

```diff
@@ -0,0 +1,98 @@
+from unittest import TestCase
+
+import pytest
+
+from miio import DreameVacuum
+from miio.tests.dummies import DummyMiotDevice
+
+from ..dreamevacuum_miot import (
+    DREAME_F9,
+    ChargingState,
+    CleaningModeDreameF9,
+    DeviceStatus,
+    FaultStatus,
+    OperatingMode,
+    WaterFlow,
+)
+
+_INITIAL_STATE = {
+    "battery_level": 42,
```

Comment:
Would you mind combining all the tests under the existing test file? See #1287's `miio/tests/test_airpurifier_miot.py` for a nice example.

Also, the "state" should be a representation of the device response, i.e., there should be no enums in those values while the property getters should be checked to return the correct information.